src/main/java/org/jabref/gui/preferences/XmpPrefsTab.java

```diff
@@ -174,4 +138,30 @@ public boolean validateSettings() {
     public String getTabName() {
         return Localization.lang("XMP-metadata");
     }
+
+    private class XMPPrivacyFilter {
+
+        private final SimpleStringProperty field;
```

Comment:
is it really necessary to make this a property instead of a simple get/set java prop? Moreover, I like the idea of creating a new class for this but would go one step further: use it everywhere (i.e. the properties class should set/get a list of privacy filter).

Reply:
I understand your idea but it is necessary as the cellValueFactory expects an Observable.
Another idea would be to have a converter/PropertyValueFactory. 
 lastNameCol.setCellValueFactory(new PropertyValueFactory<Person,String>("lastName"));